src/form.js

```diff
@@ -764,11 +741,13 @@ define([
             //    this.fixBrokenReferences(mug);
             //}
             if (mug.options.isODKOnly) {
-                this.updateError({
+                // is this a good candidate for "info" message level?
+                mug.addMessage(null, {
+                    key: 'form-odk-only-warning',
+                    level: 'warning',
```

Comment:
minor, but would be cool to see an ENUM like struct for the levels. especially if there is only a subset of levels that work. in the event of a typo it will throw an error